### ISSUE TITLE
Sign in button is not functional

### ISSUE THREAD

**Issue by sindresorhus** [dev-protocol/stakes.social#876]
When the user doesn't have MetaMask installed, the "Sign in" button in nav bar does nothing.

DevTools shows this error when I click the button:

```
_app-94fd960c2630a4b680ec.js:1 @metamask/detect-provider: Unable to detect window.ethereum.
```

**Comment by aggre** [comment]
@sindresorhus This issue may have been resolved by #902. Can you check again? (The PR is already deployed to https://stakes.social)

**Comment by sindresorhus** [comment]
Confirmed fixed.
